SwiftRuntimeLibrary/SwiftCore.cs

```diff
@@ -381,6 +381,9 @@ struct MetadataResponse {
 			return response.Metadata;
 		}
 
+		[DllImport (kXamGlue, EntryPoint = XamGlueConstants.SwiftCore_GetEnumMetadata)]
+		internal static unsafe extern bool GetEnumMetadataByName (byte* stringData, out SwiftMetatype swiftMetatype);
```

Comment:
You can avoid the unsafe code by doing:

```suggestion
		internal static extern bool GetEnumMetadataByName (byte[] stringData, out SwiftMetatype swiftMetatype);
```